### PR TITLE
Add an option to allow accumulating sub audio chunks for one processing

### PR DESCRIPTION
**Pull request by huningxin**
This PR contains another commit that adds an option for verbose log.

@Honry , PTAL, thanks!

### CODE REVIEW

File: js/whisper-demo/index.js [hunk 1]
```diff
@@ -34,7 +34,13 @@ function updateConfig() {
             options.dataType = pair[1];
         }
         if (pair[0] == 'maxChunkLength') {
-            options.maxChunkLength = parseFloat(pair[1]);
+            options.maxChunkLength = parseInt(pair[1]);
```
Comment (Honry) [inline js/whisper-demo/index.js]: I'd like to keep this as `parseFloat`, as we may try to reduce the `maxChunkLength` within 1 sec to get better real-time experience once we get better performance optimization.

File: js/whisper-demo/main.js [hunk 2]
```diff
@@ -345,10 +360,14 @@ async function processAudioBuffer() {
                 recognition._onresult(ret, true);
             }
```
Comment (Honry) [inline js/whisper-demo/main.js:365]: If it is `accumulateSubChunks` mode, `subText` should be refreshed every time until the `subAudioChunks` is empty.

File: js/whisper-demo/index.js (hunk 1 above)
Comment (huningxin) [reply js/whisper-demo/index.js]: Makes sense

File: js/whisper-demo/main.js (hunk 2 above)
Comment (huningxin) [reply js/whisper-demo/main.js:365]: yes, good point, fixed in https://github.com/Honry/onnxruntime-inference-examples/pull/3/commits/c094335b2ca39521a04c870369a16eb686159c7a

File: js/whisper-demo/index.js (hunk 1 above)
Comment (huningxin) [reply js/whisper-demo/index.js]: Reverted in latest commit.

**Review by Honry — APPROVED** [review]
LGTM, thanks!

**Pull request merged**